spec/factories/authors.rb

```diff
@@ -0,0 +1,17 @@
+FactoryGirl.define do
+  factory :author do
+    name "MyString"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.